scenarios/README.md

```diff
@@ -0,0 +1,82 @@
+
+## 2014-E&S
+Summary: Transition scenarios for evaluation groups from
+the 2014 E&S Study from the U.S. Department of Energy.
+
+Wigeland, R., T. Taiwo, H. Ludewig, M. Todosow, W. Halsey, J. Gehin, R. Jubin, J. Buelt,
+S. Stockinger, K. Jenni, B. Oakley (2014), “Nuclear Fuel Cycle Evaluation and Screening – Final
+Report,” FCRD-FCO-2014-000106, Fuel Cycle Technologies Programme.
+
+## 2017-bae-europe
+Summary: French fleet simulation with random lifetime
+extension and sensitivity analysis.
+
+Related to:
+
+  Bae, Jin Whan. Fuel Cycle Transition Simulation Capabilities in CYCLUS. 2018. University of Illinois at Urbana-Champaign, MS Thesis, https://www.ideals.illinois.edu/handle/2142/102517. IDEALS, http://hdl.handle.net/2142/102517.
+
+
+
+## aqueous_reprocess
+Summary: A simulation where uox_waste is output as
+reprocess_waste, uox_U and uox_Pu to get familiar
+with the separations cycamore archetype. It
+contains a source, separations and sink facility.
+
+Related to:
+
+  Chee, Gwendolyn J. Sensitivity Analysis of Nuclear Fuel Cycle Transitions. 2019. University of Illinois at Urbana-Champaign, MS Thesis, https://www.ideals.illinois.edu/handle/2142/106272.
+
+
+
+## eu_1950-2020
+Summary: A project simulating the number of reactors
+and net capacity of a hypothetical system.
+
+Related to:
+
+  Bae, Jin Whan. Fuel Cycle Transition Simulation Capabilities in CYCLUS. 2018. University of Illinois at Urbana-Champaign, MS Thesis, https://www.ideals.illinois.edu/handle/2142/102517. IDEALS, http://hdl.handle.net/2142/102517.
+
+
+
+## EU_1950_to_2020_reprocess
+Summary: A project building off of eu_1950_2020 to
+simulate the number of reactors and net capacity of
+a hypothetical system with reprocessing capabilities.
+
+Related to:
+
+  Bae, Jin Whan. Fuel Cycle Transition Simulation Capabilities in CYCLUS. 2018. University of Illinois at Urbana-Champaign, MS Thesis, https://www.ideals.illinois.edu/handle/2142/102517. IDEALS, http://hdl.handle.net/2142/102517.
+
+
+
+## haleu
+Summary: This directory contains the input files and
+analysis of multiple transition scenarios to advanced
+reactors fueled by High Assay Low Enriched Uranium
+(HALEU).
+
+
+## predicting-the-past
+Summary: This directory contains the input files and
+analysis of multiple scenarios of european systems and
+United States validations.
+
+Related to:
+
+  Bae, Jin Whan. Fuel Cycle Transition Simulation Capabilities in CYCLUS. 2018. University of Illinois at Urbana-Champaign, MS Thesis, https://www.ideals.illinois.edu/handle/2142/102517. IDEALS, http://hdl.handle.net/2142/102517.
+
+
+
+## standardized-verification
+Summary: This input file simulates the paper Standards
+Verification of Fuel Cycle Modeling by Feng et al. (2016)
+
+Related to:
+
+  Feng, B., et al. “Standardized Verification of Fuel Cycle Modeling.” Annals of Nuclear Energy, vol. 94, Aug. 2016, pp. 300–12, https://doi.org/10.1016/j.anucene.2016.03.002.
+
```

Comment:
There should be a publication from Jin Whan for this, published in 2019. The paper by Bo doesn't include Cyclus.
https://www.sciencedirect.com/science/article/pii/S0306454919300179